scrapi/processing/elasticsearch.py

```diff
@@ -21,14 +21,15 @@
     # If we cant connect to elastic search dont define this class
     es = Elasticsearch(settings.ELASTIC_URI, request_timeout=settings.ELASTIC_TIMEOUT)
 
-    body = {
-        'mappings': {
-            harvester: settings.ES_SEARCH_MAPPING
-            for harvester in registry.keys()
-        }
-    }
-    es.cluster.health(wait_for_status='yellow')
-    es.indices.create(index=settings.ELASTIC_INDEX, body=body, ignore=400)
+    # body = {
+    #     'mappings': {
+    #         harvester: settings.ES_SEARCH_MAPPING
+    #         for harvester in registry.keys()
+    #     }
+    # }
+    # es.cluster.health(wait_for_status='yellow')
```

Comment:
Not sure what we want the new mappings to be yet. Will probably replace this with something that generates mappings from a jsonschema specification.